airgun/fixtures.py

```diff
@@ -7,13 +7,57 @@
 """
 import pytest
 
+import nailgun.entities
+from fauxfactory import gen_string
+
 from airgun.session import Session
 
 
+@pytest.fixture(scope='module')
+def module_org(request):
```

Comment:
Yes, fixture implementation seems as correct from style perspective, but not sure whether we should go away from classical `setup_*` as it seems a little bit redundant if we need to have different setups for test module or test classes in case we need only new organization, only new organization and product, new organization, product and repo and etc, so code will raise dramatically and has abnormal redundancy.
We need to use fixtures because it is 'pytestish' is not an argument